api/src/events.rs

```diff
@@ -96,6 +97,12 @@ impl EventsApi {
         limit: Query<Option<u16>>,
     ) -> BasicResultWith404<Vec<VersionedEvent>> {
         // TODO: Assert that Event represents u64s as strings.
+        event_handle.0.verify(0).map_err(|err| {
```

Comment:
Why only zero recursion on this one?

Reply:
It starts at 0, but every recursive call internally, increments by 1